components/script/dom/window.rs

```diff
@@ -871,6 +872,12 @@ impl WindowMethods for Window {
         self.ClearTimeout(handle);
     }
 
+    // https://html.spec.whatwg.org/multipage#microtask_queuing
```

Comment:
This link doesn't work; maybe we want https://html.spec.whatwg.org/multipage/#dom-queuemicrotask instead?

Reply:
Thanks for noticing. I had an underscore instead of a hyphen, but the dom-methodname link is what Servo usually has for methods so I'll make it that instead.